site/app.yaml

```diff
@@ -10,3 +10,19 @@ handlers:
 - url: /(.*)
   static_files: _site/\1
   upload: _site/(.*)
+
+skip_files:
+- ^_apps/.*
+- ^_contributors/.*
+- ^_includes/.*
+- ^_layouts/.*
+- ^_sass/.*
+- ^assets/.*
+- ^node_modules/.*
+- ^_config\.yml$
+- ^Gemfile.*
+- ^package\.json$
+- ^sw-precache-config\.js$
+- ^(.*/)?.*\.md$
+- ^(.*/)?.*\.gpg$
+- ^(.*/)?\..*$
```

Comment:
Allah Akram and you are welcome :smiley: 

I think it's fine to not include these ignore regexes because they aren't used in this repo.

I'll include them back if you wish with other tweaks:

```
---
runtime: go
api_version: go1
default_expiration: 300s

handlers:
- url: /
  static_files: _site/index.html
  upload: _site/index\.html
  secure: always
  http_headers:
    X-UA-Compatible: "IE=edge"

- url: /(.*)
  static_files: _site/\1
  upload: _site/(.*)
  secure: always

skip_files:
- ^_apps
- ^_contributors
- ^_includes
- ^_layouts
- ^_sass
- ^assets
- ^node_modules
- ^_config\.yml$
- ^Gemfile.*
- ^package\.json$
- ^sw-precache-config\.js$
- ^(.*/)?.*\.md$
- ^(.*/)?.*\.gpg$

- ^(.*/)?#.*#$
- ^(.*/)?.*~$
- ^(.*/)?.*\.py[co]$
- ^(.*/)?.*/RCS/.*$
- ^(.*/)?\..*$

```

based on https://github.com/GoogleChrome/chromium-dashboard/blob/master/app.yaml & https://github.com/h5bp/server-configs-gae/blob/master/app.yaml